index.ts

```diff
@@ -0,0 +1,14 @@
+import sequelize, { connect } from "./src/config/dbConnection";
+import app from "./src/utils/server";
+
+app.listen(5000, async () => {
```

Comment:
why hardcoded port

Reply:
let me fix that

Reply:
i have squahed commits from ch-eslint-test-#187419154 into ch-project-setup-#187419151 and closed new PR kindly review and merge accordingly , i din't update PR mssage

Reply:
again once this is merged i can continue , i still have many local chnage that i didn't add for sake of avoid overloading many chnage into this single PR